appshell/client_handler_mac.mm

```diff
@@ -56,9 +56,12 @@
   NSWindow* window = [view window];
   std::string titleStr(title);
   NSString* str = [NSString stringWithUTF8String:titleStr.c_str()];
-  [window setTitle:str];
```

Comment:
Are you able to repro this scenario? I will check with you tomorrow in the office.